### ISSUE TITLE
BooleanBlock support

### ISSUE THREAD

**Issue by jkevingutierrez** [wagtail/wagtail-localize#304]
Is there a way to have a BooleanBlock checked on the original page, but have it unchecked on some localized pages?

**Comment by kaedroho** [comment]
I think it would be good to support this, and other data block types like integer/float/date, etc as overridable fields. Thanks for the suggestion!

**Comment by jkevingutierrez** [comment]
Right, it would also be good to keep the validations in the blocks. For example, it allows modifying the URLBlock for any text

**Comment by Shaffle1** [comment]
👍🏼 
Need it right now! 😄
